communication/kubernetesio-x-guidelines.md

```diff
@@ -0,0 +1,43 @@
+* Kubernetes Slack Announcement Bot
+* Mastadon: [k8sContributors@hachiderm](https://hachyderm.io/@K8sContributors)
+* X: [k8sContributors](https://x.com/k8sContributors)
+
+### End User focused Channels are
+
+* X: [kubernetesio](https://x.com/kubernetesio)
+* [LinkedIn](https://www.linkedin.com/company/kubernetes/posts/)
+
+### Mixed Channels:
+
+* [Lwkd Newsletter](https://lwkd.info/)
+
+## The following policy will describe the usage policy of the kubernetesio X account.
+
+* The kubernetesio account is reaching hundreds of thousands people. As such, we need to make sure we link to things that are owned by the community itself (mailing lists, GitHub, surveys, etc.).
+    * As a rule, we should not link to individuals' Twitter accounts or other personal properties given the massive amount of attention it could create
+    * Retweets of long time contributors and people that can generally sustain the attention they will receive from the tweet is encouraged - this is to be decided by the [SIG ContribEx Comms Leads](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/README.md#roles) in a case by case decision
+        * In general the person should be a well known technologist or significant contributor
+        * If posting about or linking to Mastodon make sure you have the correct account as there are often many hits on a single user on other instances
+    * If you have questions, please reach out to the [SIG ContribEx Comms Leads](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/README.md#roles) or [social media team](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/role-handbooks/Social-Media.md) (leads and/or shadows)
+* The preferred way for managing tweets is handled via Buffer.
+    * Active members recognized in the [SIG ContribEx Comms readme](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/README.md) can propose tweets via Buffer
+    * [SIG ContribEx Comms Leads](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/README.md#roles) and SIG ContribEx Comms Social Media Leads can approve tweets
+    * [SIG ContribEx Comms Leads](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/README.md#roles) and SIGContribEx Comms Social Media Leads must not approve their own proposed tweets - this can be overruled by emergency posts to be defined in sensitive communication
+* Request for communication (social media, surveys, etc.) can be made via a GitHub Issue in the Community Repository or via a communication request in #sig-contribex-comms. The GitHub issue request is the preferred method.
+    * It is not possible to request communication via kubernetesio X account directly. We recommend that requesters define a targeted recipient group, rather than a target channel or account. The current recipient groups are:  \
```

Comment:
```suggestion
    * Direct communication requests to the kubernetesio X account are not accepted. Requesters should specify a targeted recipient group rather than a specific channel or account. The current recipient groups are:  \
```